Dockerfile

```
@@ -1,18 +1,47 @@
-FROM node:24.1.0-alpine3.21 AS build-stage
+FROM node:24.1.0-alpine3.21 AS builder
+
 RUN apk update && apk add git
```

Skip caching of packages index if not needed

```suggestion
RUN apk update --no-cache && apk add --no-cache git
```